### DISCUSSION TITLE
Issue with [.xltm] filename extension

### DISCUSSION THREAD

**Discussion by Rachel-0107** [microsoft-search/pnp-modern-search#2118] [Q&A]
**Version used**
the latest one

**Describe the bug**
 [xltm] file  extension do not appear in the search results.

**To Reproduce**
1. Save some files with [.xltm] file extension in Documanet Library.
2. Set a page using PnP filters and PnP results perts.
3. See a [FileType] filter.

**Expected behavior**
The FileType filter has the [.xltm] extension.

**Screenshots**
![image](https://user-images.githubusercontent.com/106294989/170418288-5613b467-181a-4de9-962c-9a157ef2854e.png)
![image](https://user-images.githubusercontent.com/106294989/170418400-0c3fa179-9784-4ef1-b8b2-4598d0a8298d.png)

**Desktop (please complete the following information):**
 - Chrome, Edge

**Comment by FranckyC** [comment]
Hi @Rachel-0107, at first glance it does not seem to be a Web Part issue. Did you test using the default search page or SharePoint Search Qurey Tool? Also can you share your Web Part configuration and especially the query template?
